packages/fxa-settings/src/components/AppLayout/index.tsx

```diff
@@ -172,14 +172,18 @@ export const AppLayout = ({
             </main>
           </div>
         )}
-      </div>
+
       {showLocaleToggle && (
-        <footer>
-          <div className="fixed bottom-6 left-6 z-10">
+          <footer
+            className="w-full py-4 mx-4 flex-wrap mobileLandscape:flex-nowrap mobileLandscape:mx-8 mobileLandscape:pb-6 flex text-grey-400"
```

Comment:
Small tweak for mobile padding (otherwise the toggle is on the edge of the screen)
```suggestion
            className="w-full py-4 px-4 flex-wrap mobileLandscape:flex-nowrap mobileLandscape:mx-8 mobileLandscape:pb-6 flex text-grey-400"
```